Subject: opensuse: add info about UEFI support
From: Pino Toscano ptoscano@redhat.com Tue Mar 3 16:45:19 2020 +0100
Date: Thu Mar 5 17:04:46 2020 +0000:
Git: 5bbe30db9c60fc1ae684b178faaefcc3888fba80

openSUSE supports UEFI on x86_64 since openSUSE 12.2:
https://en.opensuse.org/openSUSE:UEFI

Signed-off-by: Pino Toscano <ptoscano@redhat.com>

--- osinfo-db-20200214.orig/os/opensuse.org/opensuse-12.2.xml
+++ osinfo-db-20200214/os/opensuse.org/opensuse-12.2.xml
@@ -32,6 +32,8 @@
     <release-date>2012-09-05</release-date>
     <eol-date>2014-01-15</eol-date>
 
+    <firmware arch="x86_64" type="efi"/>
+
     <media arch="i686">
       <url>http://download.opensuse.org/distribution/12.2/iso/openSUSE-12.2-DVD-i586.iso</url>
       <iso>
